framework-examples/react/src/App.js

```diff
@@ -49,6 +51,7 @@ export default class App extends PureComponent {
         </FormField>
         <div style={{paddingTop: '12px'}}>
           <button onClick={() => this.setState({indeterminate: true})}>Make Indeterminate</button>
+          <button onClick={() => this.setState({disabled: !disabled})}>Disable</button>
```

Comment:
Can you change the text here to say "Toggle Disabled"? It's more indicative of what it actually does.